dashboard/src/components/resource-list/ResourceListView.js

```diff
@@ -8,6 +8,7 @@ import MenuItem from "@material-ui/core/MenuItem";
 import FormControl from "@material-ui/core/FormControl";
```

Comment:
I realized that isLoading is broken for ResourceView because it flashes the empty resource list every time